ta/os_test/os_test.c

```diff
@@ -35,6 +35,7 @@
 #include "os_test.h"
 #include "testframework.h"
 #include "test_float_subj.h"
+#include "undef_instr.h"
```

Comment:
Is that very likely as of now? Jeromes proposal doesn't clutter down that much. Also, if we in the future would get other arch's, then I guess we could just expand the `#if ... #else`?